### PR TITLE
Fix tests for ops > 1.5.2

### PR DESCRIPTION
**Pull request by mthaddon**
Operator Framework > 1.5.2 is stricter with config values, update so that tests pass.

### CODE REVIEW

**Review by arturo-seijas — DISMISSED** [review]
LGTM

File: tox.ini [hunk 1]
```diff
@@ -52,6 +52,15 @@ commands = flake8 src/ tests/ lib/
 deps =
     flake8==3.7.9
 
+[testenv:static]
```
Comment (arturo-seijas) [inline tox.ini:64]: Could you add this to envlist?

**Review by arturo-seijas — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (mthaddon) [reply tox.ini:64]: I have done now, yep.

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by gregory-schiano — APPROVED** [review]
LGTM
As discussed with mthaddon, there "Comment on PR" doesn't work, but it's an issue on workflow library

**Pull request merged**